### PR TITLE
Backend readme

### PR DESCRIPTION
**Pull request by Vayras**
This PR is focused on the updating the Readme to be backend focused only, issue #1463

### CODE REVIEW

**Comment by ecurrencyhodler** [comment]
Thanks for submitting this. Documentation looks really nice.

Would you mind adding instructions to test locally as well?

**Comment by Vayras** [comment]
> Thanks for submitting this. Documentation looks really nice.
> 
> Would you mind adding instructions to test locally as well?

yeah will do that as well, where can I find a reference for that?

**Comment by Vayras** [comment]
> > Thanks for submitting this. Documentation looks really nice.
> > Would you mind adding instructions to test locally as well?
> 
> @ecurrencyhodler  yeah will do that as well, where can I find a reference for that?

**Comment by ecurrencyhodler** [comment]
I think just direct them to install yarn. This is also what @MahtabBukhari recommended in the hunters tribe:

"Run the command 'yarn install'. 

After that, execute 'yarn jest test-file-name'. As you run the command, an error will occur in the node module file. There are three files you have to open from the link shown with the error in the terminal. After opening each file, comment out the entire content.

Once you have commented out these three files, on the fourth attempt, you should be able to run the test locally."

**Comment by Vayras** [comment]
> I think just direct them to install yarn. This is also what @MahtabBukhari recommended in the hunters tribe:
> 
> "Run the command 'yarn install'.
> 
> After that, execute 'yarn jest test-file-name'. As you run the command, an error will occur in the node module file. There are three files you have to open from the link shown with the error in the terminal. After opening each file, comment out the entire content.
> 
> Once you have commented out these three files, on the fourth attempt, you should be able to run the test locally."

Sure, Adding these instructions right away, Thanks

**Comment by Vayras** [comment]
@ecurrencyhodler quick question, as far as what I have learned we use jest for frontend testing and mockery for backend testing, so I think the instructions for setting up jest locally should be added to the frontend readme, let me know If I got that wrong.

**Comment by ecurrencyhodler** [comment]
OOo good point. You're right. Let's get a review on your instructions from raph/kev and then merge.

File: README.md [hunk 1]
```diff
@@ -1,212 +1,122 @@
...
+## Optional Features
 
-If you would like to enable Meme image upload for organization's add the meme env key and value to the .env file,
-Meme image upload works with Relay enabled, so a running Relay is required for Meme to work.
+### Redis for Caching
 
-```
-    MEME_URL=
-```
+Configure Redis by adding the `REDIS_URL` or other relevant variables to your `.env` file.
```
Comment (elraphty) [inline README.md:79]: Can you specify the other relevant variables? 
```
REDIS_HOST=
REDIS_USER=
REDIS_PASS=
REDIS_DB=
```

To make it easier for developers.

**Comment by Vayras** [comment]
@elraphty yeah , will update this

**Comment by Vayras** [comment]
@elraphty updated the readme

File: README.md [hunk 2]
```diff
@@ -78,6 +77,14 @@ go build .
 
 Configure Redis by adding the `REDIS_URL` or other relevant variables to your `.env` file.
 
+```sh
```
Comment (elraphty) [inline README.md:80]: What is the sh for?

**Comment by Vayras** [comment]
@elraphty it will format the text in a code block

**Comment by ecurrencyhodler** [comment]
good work. paid!

**Pull request merged**